packages/autocomplete-core/src/types/AutocompleteApi.ts

```diff
@@ -9,6 +10,10 @@ export interface AutocompleteScopeApi<TItem extends BaseItem>
    * Triggers a search to refresh the state.
    */
   refresh(): Promise<void>;
+  /**
+   * Functions to navigate to a URL.
+   */
+  navigator: AutocompleteNavigator<TItem>;
```

Comment:
Wondering if it's worth providing the whole props here? It feels like just passing `navigator` solves one scenario, but not others when people will introduce different plugins?

Reply:
I did debate when moving to pass only navigator that it could be interesting to pass the whole props, although as they probably would be spread, I guess it can wait until we have more need?

Reply:
We pass the `state` object without spreading so I would rather see the whole `props` object without spreading, no?

Reply:
yes, I see the argument for all of props to be passed in a single object too. I guess I was more thinking on how that's currently not an API we expose, and from now on `props` would be available too.